services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -195,22 +275,18 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
               label="Report Title and Additional Comment"
               required
               props={{ maximumCharacters: 500, rows: 3 }}
-              placeholder="Include a precise and descriptive title to the report."
               component={RenderAutoSizeField}
               validate={[required, maxLength(500)]}
             />
+            <Typography.Text className="report-instructions">
```

Comment:
What happens when you click Review & Submit?

If this is the direction that we're wanting to go with the UI, I wonder if it would be worth adding as a prop for the input. I guess CSS would be the other route; the FormWrapper should give different classes for view/edit.

Other than that, LGTM.

Reply:
I'm not sure how ubiquitous those instructions might be.  Might not be the worst idea to have them added to the fields as a prop, but I can't recall seeing designs for this kind of thing elsewhere much.

![image](https://github.com/bcgov/mds/assets/83598933/fd755ba5-7b73-44e7-9ad5-d6390d0fd6bd)
It looks the same on Review and Submit which I think is okay since it just reiterates the warning/info

Reply:
Ehhh they're not on the mockup, so I don't think they're supposed to be on the view mode.

I think it's worth checking with the ux team what their plans are.